packages/main/src/Select.ts

```diff
@@ -207,6 +207,8 @@ class Select extends UI5Element implements IFormElement {
 	/**
 	 * Defines a reference (ID or DOM element) of component's menu of options
 	 * as alternative to define the select's dropdown.
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to use as menu.
```

Comment:
same:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the menu.